sheetstorm/settings/development.py

```diff
@@ -14,3 +14,5 @@
 CAPTCHA_TEST_MODE = True
 
 VALID_EMAIL_DOMAIN_LIST = ["codepoets.it"]
+
+EMAIL_SIGNUP_VERIFICATION = False
```

Comment:
Looks like a separate feature, do we really want it this way in production?

Reply:
I agree with @rwrzesien. Eveit if it was to say I suggest we should have it `True` by default.

Reply:
But I thought these are development-only settings? This setting is already introduced in `base.py` with default `True` value, therefore the production shouldn't be affected?
Or did I miss something? 😕

Reply:
@rwrzesien @dybi this setting is set in `development.py`. On server we have `local_settings.py` which inherits from `production.py` which inherits from `base.py`. There is no chance that we use setting which is set in this file. However I think that in `base.py` should be this set on `None` or even commented to enforce someone who wants to run server see what setting is really valid for him, but if you guys wants to have `True` as a value in `base.py` I would fight with this decision :). In this file for me everything is fine.

Reply:
What I have meant is that we were not sending emails at all (I think so?) and now we are adding this functionality in not really related pull request. So the question was rather - do we want this feature? If yes than maybe it would be easier to provide it in separate pull request.

Reply:
@rwrzesien The emails are sent to the console output, from what I was told. The activation proces still applies anyway, so I need this setting to bypass this. Otherwise I can't activate an account during tests without fiddling with database, which was undesirable in e2e.